src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_DashboardStack/Dashboard_RootScreen/components/Announcement.tsx

```diff
@@ -5,71 +5,134 @@ import {
   getTextForLanguage,
   useTranslation,
 } from '@atb/translations';
-import {Image, View} from 'react-native';
+import {Image, Linking, StyleProp, View, ViewStyle} from 'react-native';
 import {StyleSheet, useTheme} from '@atb/theme';
 import {ThemeIcon} from '@atb/components/theme-icon';
 import {Close} from '@atb/assets/svg/mono-icons/actions';
 import {PressableOpacity} from '@atb/components/pressable-opacity';
 import {insets} from '@atb/utils/insets';
+import {
+  GenericSectionItem,
+  LinkSectionItem,
+  Section,
+} from '@atb/components/sections';
+import {AnnouncementSheet} from './AnnouncementSheet';
+import {useBottomSheet} from '@atb/components/bottom-sheet';
+import {animateNextChange} from '@atb/utils/animation';
+import {useAnalytics} from '@atb/analytics';
+import {useAnnouncementsState} from '@atb/announcements';
+import Bugsnag from '@bugsnag/react-native';
 
 type Props = {
   announcement: AnnouncementType;
-  onDismiss: (announcement: AnnouncementType) => void;
+  style?: StyleProp<ViewStyle>;
 };
 
-export const Announcement = ({announcement, onDismiss}: Props) => {
-  const style = useStyle();
+export const Announcement = ({announcement, style}: Props) => {
+  const styles = useStyle();
   const {t} = useTranslation();
   const {language} = useTranslation();
   const {theme} = useTheme();
+  const analytics = useAnalytics();
+  const {dismissAnnouncement} = useAnnouncementsState();
+  const {open: openBottomSheet, close: closeBottomSheet} = useBottomSheet();
+
+  const handleDismiss = (announcement: AnnouncementType) => {
```

Comment:
Now that `handleDismiss` is inside Announcement, `announcement` actually doesn't have to be a parameter and can instead be used directly.